modules/deployment/src/test/java/org/mule/runtime/module/deployment/logging/LoggingAppStartErrorTestCase.java

```diff
@@ -0,0 +1,176 @@
+
+  private static final int PROBER_TIMEOUT = 1000;
+  private static final int PROBER_INTERVAL = 100;
+
+  private DefaultMuleApplication application;
+  private ArtifactContext mockArtifactContext;
+
+  private final File appLocation = new File("fakeLocation");
+
+  TestLogger loggerDefaultMuleApplication = getTestLogger(DefaultMuleApplication.class);
+  TestLogger loggerDefaultArtifactDeployer = getTestLogger(DefaultArtifactDeployer.class);
+
+  @Parameterized.Parameters(name = "Parallel: {0}")
+  public static List<Boolean> params() {
+    return asList(false);
+  }
+
+  public LoggingAppStartErrorTestCase(boolean parallelDeployment) {
+    super(parallelDeployment);
+  }
+
+  @Before
+  public void doSetUp() throws Exception {
+    MuleApplicationClassLoader parentArtifactClassLoader = mock(MuleApplicationClassLoader.class);
+    mockArtifactContext = mock(ArtifactContext.class);
+    MuleContextWithRegistry muleContext = mock(MuleContextWithRegistry.class);
+    when(mockArtifactContext.getMuleContext()).thenReturn(muleContext);
+
+    DefaultRegistry defaultRegistryMock = mock(DefaultRegistry.class);
+    when(mockArtifactContext.getRegistry()).thenReturn(defaultRegistryMock);
+
+    NotificationListenerRegistry notificationListenerRegistryMock = mock(NotificationListenerRegistry.class);
+    when(defaultRegistryMock.lookupByType(any())).thenReturn(Optional.of(notificationListenerRegistryMock));
+
+    ApplicationDescriptor applicationDescriptorMock = mock(ApplicationDescriptor.class, RETURNS_DEEP_STUBS.get());
+    when(applicationDescriptorMock.getClassLoaderModel())
+        .thenReturn(new ClassLoaderModel.ClassLoaderModelBuilder().containing(new URL("file:/target/classes")).build());
+    application = new DefaultMuleApplication(applicationDescriptorMock, parentArtifactClassLoader, emptyList(),
+                                             null, mock(ServiceRepository.class),
+                                             mock(ExtensionModelLoaderRepository.class),
+                                             appLocation, null, null,
+                                             getRuntimeLockFactory(),
+                                             mock(MemoryManagementService.class),
+                                             mock(ArtifactConfigurationProcessor.class));
+
+    Method setArtifactContext = application.getClass().getDeclaredMethod("setArtifactContext", ArtifactContext.class);
+    setArtifactContext.setAccessible(true);
+    setArtifactContext.invoke(application, mockArtifactContext);
+  }
+
+  @Test
+  public void whenAppThrowsLifecycleExceptionWhileStartingTheErrorShouldBeLogged() throws Exception {
+    MuleContext mockedMuleContext = mock(MuleContext.class);
+    when(mockArtifactContext.getMuleContext()).thenReturn(mockedMuleContext);
+    String expectedLogMessage = "Could not start HTTP server for 'lisConfig' on port 27664: Address already in use";
```

Comment:
make this message something that explicitely indicates what is happening instead of this http-ish error message.